lib/galaxy/tool_util/verify/interactor.py

```diff
@@ -102,12 +102,19 @@ class GalaxyInteractorApi(object):
     def __init__(self, **kwds):
         self.api_url = "%s/api" % kwds["galaxy_url"].rstrip("/")
         self.master_api_key = kwds["master_api_key"]
+        self.verify = not kwds.get('noverify', False)
+        if not self.verify:
+            # disable the following warning:
+            # Unverified HTTPS request is being made.
+            # Adding certificate verification is strongly advised.
+            # See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
+            import urllib3
+            urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
A thought: does this disable the warnings for the whole Galaxy app? Maybe we can live with the warnings?

Reply:
I don't know. I think it will disable only the given warning but I have no idea about the scope of this.
Tha said, the warning is quite ugly on command line:

> /softs/bioinfo/galaxy-dev/.venv/lib/python2.7/site-packages/urllib3/connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
>   InsecureRequestWarning)